### ISSUE TITLE
Blob storage issue

### ISSUE THREAD

**Issue by vdesabou** [vdesabou/kafka-docker-playground#14]
```
[2020-03-10 13:03:05,140] ERROR WorkerSinkTask{id=azure-blob-sink-0} Task threw an uncaught and unrecoverable exception (org.apache.kafka.connect.runtime.WorkerTask)
org.apache.kafka.connect.errors.ConnectException: org.apache.kafka.connect.errors.ConnectException: java.lang.reflect.InvocationTargetException
        at io.confluent.connect.azure.blob.AzureBlobStorageSinkTask.start(AzureBlobStorageSinkTask.java:125)
        at org.apache.kafka.connect.runtime.WorkerSinkTask.initializeAndStart(WorkerSinkTask.java:301)
        at org.apache.kafka.connect.runtime.WorkerSinkTask.execute(WorkerSinkTask.java:189)
        at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:177)
        at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:227)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.kafka.connect.errors.ConnectException: java.lang.reflect.InvocationTargetException
        at io.confluent.connect.storage.StorageFactory.createStorage(StorageFactory.java:55)
        at io.confluent.connect.azure.blob.AzureBlobStorageSinkTask.start(AzureBlobStorageSinkTask.java:104)
        ... 9 more
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at io.confluent.connect.storage.StorageFactory.createStorage(StorageFactory.java:50)
        ... 10 more
Caused by: java.lang.NoClassDefFoundError: io/netty/handler/codec/http/QueryStringDecoder
        at com.microsoft.azure.storage.blob.SharedKeyCredentials.getCanonicalizedResource(SharedKeyCredentials.java:171)
        at com.microsoft.azure.storage.blob.SharedKeyCredentials.buildStringToSign(SharedKeyCredentials.java:105)
        at com.microsoft.azure.storage.blob.SharedKeyCredentials.access$0(SharedKeyCredentials.java:85)
        at com.microsoft.azure.storage.blob.SharedKeyCredentials$SharedKeyCredentialsPolicy.sendAsync(SharedKeyCredentials.java:264)
        at com.microsoft.azure.storage.blob.RequestRetryFactory$RequestRetryPolicy.attemptAsync(RequestRetryFactory.java:155)
        at com.microsoft.azure.storage.blob.RequestRetryFactory$RequestRetryPolicy.sendAsync(RequestRetryFactory.java:75)
        at com.microsoft.azure.storage.blob.RequestIDFactory$RequestIDPolicy.sendAsync(RequestIDFactory.java:60)
        at com.microsoft.azure.storage.blob.TelemetryFactory$TelemetryPolicy.sendAsync(TelemetryFactory.java:70)
        at com.microsoft.rest.v2.http.HttpPipeline.sendRequestAsync(HttpPipeline.java:67)
        at com.microsoft.rest.v2.RestProxy.sendHttpRequestAsync(RestProxy.java:99)
        at com.microsoft.rest.v2.RestProxy.invoke(RestProxy.java:126)
        at com.microsoft.azure.storage.blob.$Proxy41.create(Unknown Source)
        at com.microsoft.azure.storage.blob.GeneratedContainers.createWithRestResponseAsync(GeneratedContainers.java:215)
        at com.microsoft.azure.storage.blob.ContainerURL.create(ContainerURL.java:208)
        at com.microsoft.azure.storage.blob.ContainerURL.create(ContainerURL.java:178)
        at io.confluent.connect.azure.blob.storage.AzureBlobStorage.createContainerIfNotExists(AzureBlobStorage.java:91)
        at io.confluent.connect.azure.blob.storage.AzureBlobStorage.<init>(AzureBlobStorage.java:78)
        ... 15 more
Caused by: java.lang.ClassNotFoundException: io.netty.handler.codec.http.QueryStringDecoder
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at org.apache.kafka.connect.runtime.isolation.PluginClassLoader.loadClass(PluginClassLoader.java:104)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 32 more
```

See https://travis-ci.com/vdesabou/kafka-docker-playground/jobs/295621528

**Comment by vdesabou** [comment]
Happening since connector version is 1.3.0 (was 1.1.3 previously used)

**Comment by vdesabou** [comment]
That is fixed in 1.3.1
